codec-http2/src/main/java/io/netty/handler/codec/http2/Http2SecurityUtil.java

```diff
@@ -49,21 +49,18 @@
      */
     private static final List<String> CIPHERS_JAVA_MOZILLA_MODERN_SECURITY = Collections.unmodifiableList(Arrays
             .asList(
+            /* openssl = ECDHE-ECDSA-AES128-GCM-SHA256 */
+            "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+            /* openssl = ECDHE-RSA-AES128-GCM-SHA256 */
+            "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
```

Comment:
It seems we should keep the REQUIRED BY HTTP/2 SPEC comment.

Reply:
Thanks. Done.